ui/app/mainui/activitycenter/controls/ReplyBadge.qml

```diff
@@ -15,32 +15,44 @@ Badge {
 
     signal replyClicked()
 
-    SVGImage {
-        id: replyIcon
-        anchors.left: parent.left
-        anchors.leftMargin: Style.current.smallPadding
-        anchors.verticalCenter: parent.verticalCenter
-        width: 16
-        height: width
-        source: Style.svg("reply-small-arrow")
-    }
+    implicitWidth: layout.implicitWidth + layout.anchors.leftMargin + layout.anchors.rightMargin
+    implicitHeight: layout.implicitHeight + layout.anchors.topMargin + layout.anchors.bottomMargin
+
+    RowLayout {
+        id: layout
+
+        anchors {
+            fill: parent
+            leftMargin: 8
+            rightMargin: 8
+            topMargin: 3
+            bottomMargin: 3
+        }
 
-    StatusBaseText {
-        anchors.left: replyIcon.right
-        anchors.leftMargin: 4
-        anchors.verticalCenter: parent.verticalCenter
-        width: Math.min(implicitWidth, 300)
-        text: repliedMessageContent
-        maximumLineCount: 1
-        elide: Text.ElideRight
-        font.pixelSize: 13
-
-        MouseArea {
-            id: replyArea
-            hoverEnabled: true
-            anchors.fill: parent
-            cursorShape: Qt.PointingHandCursor
-            onClicked: root.replyClicked()
+        spacing: 4
+
+        SVGImage {
+            id: replyIcon
+            source: Style.svg("reply-small-arrow")
+            Layout.preferredWidth: 16
+            Layout.preferredHeight: 16
+        }
+
+        StatusBaseText {
+            text: repliedMessageContent
+            maximumLineCount: 1
+            wrapMode: Text.Wrap
```

Comment:
@caybro elide working if only wrapMode != Text.NoWrap

Reply:
Of course :)

Reply:
You can actually remove the `wrapMode: Text.Wrap` if you have only 1 line; the fact it works now is because you're inside a RowLayout and given some width

Reply:
> You can actually remove the `wrapMode: Text.Wrap` if you have only 1 line; the fact it works now is because you're inside a RowLayout and given some width

I tried but without wrapMode it's not working

Reply:
That is weird... it shouldn't be that way; I'll give it a try myself